docs/4. Product Features/03. Flows/2. Flow Actions/03. Send the contact an interactive message.md

```
@@ -145,3 +145,58 @@ See the <a href="https://drive.google.com/file/d/1uiJQWIi-6U7YZu28OggvXSqZ57FYNu
+
+Step 3: Click on `Save`
+
+Step 4: On the flow editor, navigate to the languages on the top right corner.
+
+<img width="607" height="393" alt="Screenshot 2025-10-29 at 8 54 04 PM" src="https://github.com/user-attachments/assets/38dd4db5-db74-4b76-a9f0-a2198b16c89f" />
+
+Step 5: Select the language you wish to add translations for. The below screenshot shows the `Hindi` language view.
+
+<img width="596" height="381" alt="Screenshot 2025-10-29 at 8 54 34 PM" src="https://github.com/user-attachments/assets/77a8361e-80ce-4336-b17f-3ff7b589718e" />
+
+Note: The values entered under validations such as `has only the phrase`, `has a number`, etc. act as rules to verify whether a user’s response matches the expected input. These are grouped into categories, as shown below. Depending on the design, these categories may or may not be visible to the end user. Learn more about `Wait for Response` [here](https://glific.github.io/docs/docs/Product%20Features/Flows/Flow%20Actions/Wait%20for%20the%20contact%20to%20respond).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)